doc/classes/Tween.xml

```diff
@@ -137,6 +137,12 @@
 				Returns the number of remaining loops for this [Tween] (see [method set_loops]). A return value of [code]-1[/code] indicates an infinitely looping [Tween], and a return value of [code]0[/code] indicates that the [Tween] has already finished.
 			</description>
 		</method>
+		<method name="get_step_count" qualifiers="const">
+			<return type="int" />
+			<description>
+				Returns the total number of steps in this [Tween]. One step is either a single [Tweener] or a group of [Tweener]s running in parallel.
```

Comment:
Note that SubtweenTweener counts as single step. Maybe it should be clarified here.